### ISSUE TITLE
[Feature] Allow Changing Font Size on CardFormView

### ISSUE THREAD

**Issue by Cavallando** [stripe/stripe-ios#1933]
## Is your feature request related to a problem? Please describe.

We are currently using the `CardFormView` in our application but have found the customization needs to be lacking, compared with the `PaymentCardTextField`. Specifically we'd love the ability to increase/decrease font size. There is an existing [issue](https://github.com/stripe/stripe-ios/issues/1932) that details the need for a broader set of customization features but I wanted to open up a specific feature request for font size if it helps keep PRs small.

## Describe the solution you'd like

Working with the implementation, it would be nice to have the fontSize/Family be independent of the `.standard` `.borderless` styles for the container and have the fontSize/Family propagate to the underlying field components within the CardForm.

## Describe alternatives you've considered

Alternatively, we _could_ use the `PaymentCardTextField` but looking at the swift code for `CardFormView` it feels like it wouldn't be too difficult to have this supported within the form, especially since this form fits better with our design system.

## Additional context

We're currently using Stripe's React Native SDK but obviously in order to get that supported the native SDK's need to be updated first. I opened this issue here because I have more knowledge of the iOS ecosystem than I do Android's. I am happy to do this work for Android/iOS and React Native code bases, assuming it doesn't conflict with Stripe's roadmap 😄

**Comment by yuki-stripe** [comment]
Hello @Cavallando,

I've added this to our internal tracker alongside the other ticket you mentioned, but note that feature development on CardFormView isn't on our immediate roadmap. 

We're super focused on getting `PaymentSheet` to support more use cases and customization, however. We're working on this exact font size customization now, in fact!  I'd love to hear more about what drove you to choose `CardFormView`? Feel free to send me an email ([yuki@stripe.com](mailto:yuki@stripe.com)) if you don't want to share publicly.

**Comment by Cavallando** [comment]
Hey @yuki-stripe thanks for getting back to me!

I think we're in a more unique case but we're currently a part of the Stripe Orders v2 Beta. While implementing our own custom checkout with Orders API implemented in our backend we actually really wanted to use the PaymentSheet because it pretty much gave us what we needed but ran into some issues that we've discussed with the Orders beta team at Stripe.

The biggest issue for us was that you need a paymentIntentClientSecret which we wouldn't be able to get until after an Order was submitted. They mentioned that they've updated the Payment Element to use an orderClientSecret that gets returned on create so I think the inclusion of that would really help in the Mobile SDKs for the future of that feature.

Separately from the Orders API, I tried to use the customFlow param but found that we were limited in the actual PaymentMethod that was chosen. For us, ideally we'd be able to select a PaymentMethod in the PaymentSheet and it would return a paymentMethodId to be used in more custom backend implementations, but it appears right now the PaymentSheet only uses a PaymentOption with a label, image attributes. Whereas with the CardForm, we can call `createPaymentMethod` which uses the card info and returns a paymentMethodID for us to use in requests to our own API and further Stripe Orders Beta API.

So in short, we were more driven to use the CardForm because we _had_ to, but we really wanted to get PaymentSheet to work!

Hope this helps!

**Comment by sfriedman-stripe** [comment]
Hi @Cavallando, we will not be adding further functionality to STPCardFormView or STPPaymentCardTextField. We encourage you to check out our modern replacement, the [Mobile Payment Element](https://stripe.com/docs/payments/accept-a-payment?platform=ios), and send us any feedback you have! Thank you
